RWFramework/RWFramework/Playlist/Playlist.swift

```diff
@@ -77,7 +77,7 @@ class Playlist {
         audioMixer.distanceAttenuationParameters.rolloffFactor = 0.00001
         audioMixer.distanceAttenuationParameters.referenceDistance = 1
         audioMixer.distanceAttenuationParameters.maximumDistance = 200_000
-        audioMixer.renderingAlgorithm = .soundField
+        audioMixer.renderingAlgorithm = .HRTFHQ
```

Comment:
This PR works fine for me except the `HRTFHQ` rendering doesn't seem to work in the simulator (😭 ) but if it works on device, then this is all set.